src/internal/core.ts

```diff
@@ -504,7 +504,7 @@ export const YieldableError: new(
   message?: string,
   options?: ErrorOptions
 ) => Cause.YieldableError = (function() {
-  class YieldableError extends globalThis.Error {
+  class YieldableError extends (globalThis.Error as any) {
```

Comment:
Otherwise it fails with `'YieldableError' does not override 'message'` or something like that.